All EIPIP Meetings/Meeting 012.md

```diff
@@ -0,0 +1,364 @@
+
+(Matt) - I mean, my general thoughts right now is that we are trying to scale and EIP editor list of about 8 to maybe the teens or 20 tops, so I don't think too much process is needed. I think a majority or super majority is fine, I don't know how often the EIP editors should meet. Maybe they should meet quarterly to discuss if people should be removed or not
+
+(James) - Ah yeah, that's a good idea. We should add that. I'd be happy to host that. Yeah, so attendance to the meetings is actually a good requirement.
+
+(Matt) - One last thing, if you all have any ideas on requirements please message us that directly on the EIP validator. I'm trying to create a list of things that need to be implemented.
+
+(James) - I have a document where I wrote out a few of them; do you know where that is Pooja?
+
+(Pooja) - I think it's in 2 blues, maybe it's linked
+
+(James) - on the notes for the previous meeting?
+
+(pooja) - No, on today's agenda item number 3
+
+(James) - Oh yeah
+
+(James) - I just put it in the notes, let's schedule something to go over that because I think that'd be a good call to have.
+
+(Edson) - One last comment: I was thinking about this when I wrote the article for the road-map. Say because we are currently still editing the process in the EIPs themselves. If we were to onboard people now, they would be learning the old process, and they'll have to learn the new process very quickly so they'd be doing one thing and then doing another. It might be better to wait until the new process is flushed out and matured before we have people learn how to become an editor and be onboarded was my first thought.
+
+(James) - Yeah, that's a good point.
+
+(Pooja) - I think this will take some time to shape up properly so by the time we are into the new process this thing should be ready. I am hoping it would be ready publicly and it wouldn't be a problem for EIP editors. Moreover the people we are trying to consider here as an editor are actively invovled so it would be a good transition for them also to be onboarded for this process. That's just my thought, though.
+
+(James) - Mika for example would be totally fine. Being able to handle that, Matt would be totally fine to be able to handle that. The new statuses are I think getting pretty solid. Those are the, I think, the biggest ones. It'd be good to have some of that be settled before we really push through this, that's a good point so thanks for bringing that up Edson.
+## 3.1. Decisions
+
+- *12.1*- Edson will make a survey to the current editors on the expectations from an editor (time-commitment, qualifications)
+- *12.2*- James wil look into the necessary permissions and how to setup users with triaging capabilities
+- *12.3* - A process for removing editors and criteria for removing editors is a future item to discuss
+
+# 4. Separation of EIP process and hard-fork coordination

```

Comment:
replace # 4. with # 2.

# 2. Separation of EIP process and hard-fork coordination